src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@essent/nativescript-oauth2",
-  "version": "1.0.0",
+  "version": "1.4.0",
```

Comment:
I forgot to merge the last time when a pull request is approved (version 1.3.0), so that is why there is a difference in this version numbers.